lib/snippets.js

```diff
@@ -122,7 +122,7 @@ export const get${pluralize(
     )}ByIds = async ( ids, filterByFormula = '', sort = []
 ) => {
   let formula = ${'`OR(${'}ids.reduce(
-    (f, id) => ${"`${f} {ID}='${id}',`"},
+    (f, id) => ${"`${f} RECORD_ID()='${id}',`"},
```

Comment:
Makes sense + looks good! And you tested this, right?

Reply:
Yes, tested this one (directly from the issue suggestion) (":